scripts/marketing_data_test.py

```diff
@@ -375,6 +398,9 @@ def marketing_dml_test(
                     for score in fold_scores:
                         output_results.write(f"{score},")
 
+                if isinstance(est.score_, float):
+                    est.score_ = [est.score_]
```

Comment:
This is the bug where `est.score_` is float for my client sample data. So I am converting to list if it's a float.